### ISSUE TITLE
Members Monitor node results in TypeError

### ISSUE THREAD

**Issue by Eeems** [madmax3365/node-red-node-discord#2]
I'm getting the following error when trying to use the Members Monitor node
```
TypeError: Cannot read property 'name' of null
    at _loop_1 (/data/node_modules/node-red-node-discord/dist/lib/MemberMonitor.js:77:47)
    at MemberMonitor.get [as textChannelMetric] (/data/node_modules/node-red-node-discord/dist/lib/MemberMonitor.js:88:21)
    at Object._inputCallback (/data/node_modules/node-red-node-discord/dist/nodes/members-monitoring.js:28:66)
    at Object.Node._emitInput (/usr/src/node-red/node_modules/@node-red/runtime/lib/nodes/Node.js:200:18)
    at Immediate._onImmediate (/usr/src/node-red/node_modules/@node-red/runtime/lib/nodes/Node.js:179:33)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

**Comment by madmax3365** [comment]
Can you please share your `node-red` flow with me?

**Comment by Eeems** [comment]
```
[{"id":"e61d0119.565","type":"inject","z":"21b00103.816bae","name":" ","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":70,"y":100,"wires":[["dacf52d1.ea1b88"]]},{"id":"dacf52d1.ea1b88","type":"discord-monitor-members","z":"21b00103.816bae","name":"","token":"","x":250,"y":60,"wires":[["ddf8e500.a46468"]]},{"id":"ddf8e500.a46468","type":"debug","z":"21b00103.816bae","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":430,"y":60,"wires":[]}]
```

**Comment by Eeems** [comment]
I'm no longer experiencing this. Likely it was fixed by 63c0813
